content/tutorials/geodesic-getting-started.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Getting started with Geodesic"
+description: "Learn what Geodesic is and how you can start using it to simplify your local infrastructure development."
+weight: 1
+---
+
+In the landscape of developing infrastructure, there are dozens of tools that we all need on our personal machines to do our jobs. In SweetOps, instead of having you install each tool individually, we use Docker to package all of these tools into one convenient image that you can use as your infrastructure automation toolbox. We call it [Geodesic]({{< relref "reference/tools.md#geodesic" >}}) and we use it as our DevOps automation shell and as the base Docker image for all of our DevOps scripting / CI jobs.
+
+In this tutorial, we'll walk you through how to use Geodesic to execute an authenticated AWS CLI command and talk about what is going on under the hood.
```

Comment:
I wanted to get this tutorial up so that we could start working collaboratively on this, but I'm not 100% sure I picked the most useful topic (i.e. use the AWS CLI). I could see it being more useful to apply a single resource Terraform project and then destroy it, but figured we should discuss before I redid some of this. I'm also open to any other simple, concrete example which shows the utility of Geodesic without being overly arduous.

Reply:
@osterman would like to get your thoughts here. Maybe this is our MVP and we circle back to a more complicated, real world use-case tutorial in the future?